Misc/NEWS.d/next/Security/2025-01-14-11-19-07.gh-issue-128840.M1doZW.rst

```diff
@@ -0,0 +1,3 @@
+Limit the number of splitting on colons (``:``) that will occur while parsing
```

Comment:
This still mentions an implementation detail.

Also, what is the denial-of-service? Do you have a reproducer?

To me, it looks like a tiny performance improvement (which may even not be noticeable). Since it is so simple, it will not harm to backport it to maintained versions (although we are not obliged to do this), but this is far from security issue. The NEWS entry makes it looking much more severe than it is.

Reply:
Pushed an update to the newsfragment: https://github.com/python/cpython/pull/128841/commits/e1e69174a279157e0be73b045ae302381b954e4e I also tagged you into the PSRT thread, Django has treated this as a security fix, I agree that it's low severity.